src/main/java/pl/agh/edu/master_diet/repository/RecentProductRepository.java

```diff
@@ -20,6 +20,11 @@
             "ORDER BY RP.mealTime")
     List<RecentProduct> findByUserAndMealTimeDate(Long userId, LocalDate date);
 
+    @Query("SELECT SUM(RP.product.calories * (RP.portion / 100.0) * RP.amount) FROM RecentProduct RP " +
```

Comment:
Zastanowiłbym się, czy nie warto zacząć używać CriteriaApi zamiast sqli zaleznych od dialektu, ale to na potem mozna :smile:

Reply:
Tutaj używamy JPQL, a wydaje mi się że one są niezależne od dialektu

Reply:
racja, to git